graph/src/data/graphql/effort.rs

```diff
@@ -389,18 +394,24 @@ impl LoadManager {
             if overloaded {
                 kill_rate = (kill_rate + KILL_RATE_STEP * (1.0 - kill_rate)).min(1.0);
             } else {
-                kill_rate = (kill_rate - KILL_RATE_STEP).max(0.0);
+                kill_rate = (kill_rate - 2.0 * KILL_RATE_STEP).max(0.0);
```

Comment:
Do we want to document somewhere that the rate of decay is `2*KILL_RATE_STEP`? Maybe make this its own constant?

Reply:
There's a whole section of the `graph-node` manual that needs to be written about load management and how to use and tune it; I'll try and write some of that up next week. I don't want to bug people with another tuneable, since I view tuneables largely as us saying "we couldn't figure it out so we'll have you do that". But I am also not sure whether stepping down at twice `KILL_RATE_STEP` is the right way to go anyway - it's a tradeoff between stopping to drop queries quickly when an overload is brief vs settling on the 'right' kill rate when the overload is prolonged. Thinking about this more, I wonder if stepping down too quickly could be problematic during the latter.

Reply:
I wasn't necessarily suggesting to make this configurable, just to extract this to a constant so you can have a name and a comment for it.

Reply:
I totally misunderstood - I turned that factor into a constant, pulled all the constants used in this calculation into `update_kill_rate` and added comments about the calculations in the code.